community-builder packages: Add mosh

matthiasbeyer opened the pull request:
I would like to propose to add `mosh` ([mosh.org](https://mosh.org/)) to the community builder.

I like to keep my ssh sessions via suspends, and this helps.

matthiasbeyer commented:
Ah, well... this would also require some ports to be opened... and you might not want that?

Mic92 commented on modules/nixos/community-builder/packages.nix:
```diff
@@ -4,6 +4,7 @@
   environment.systemPackages = [
     pkgs.fd
     pkgs.git
+    pkgs.mosh
```
Should be fine:

```
  networking.firewall.allowedUDPPortRanges = [
    # Mosh
    {
      from = 60000;
      to = 60010;
    }
  ];
```

matthiasbeyer commented:
Added a patch for the UDP port range required for mosh.

1. This should probably go in another file, please point me to it
2. I added the range 60000-61000, not as @Mic92 requested 60000-60010 - I added the range that mosh might use. Is that okay or shall I restrict the range to 10/11 ports?

zowoq commented:
> This should probably go in another file, please point me to it

This file is fine.
